packages/story-editor/src/components/panels/document/taxonomies/taxonomies.js

```diff
@@ -26,6 +26,8 @@ import { SimplePanel } from '../../panel';
 import HierarchicalTermSelector from './HierarchicalTermSelector';
 import FlatTermSelector from './FlatTermSelector';
 
+const ENABLED_TAXONOMIES = ['story-tags', 'story-categories'];
```

Comment:
Why is this hardcoded?

Reply:
We're getting 4 taxonomies back on the story: `story-tags`, `story-verticals`, `story-colors, and `story-categories`. I added this here to make sure we only render UI for tags and categories.

Reply:
Right. IIRC we do it this way while UX is being finalized. But we definitely need to remove this eventually. So if you could add a todo comment, that‘d be good.

Reply:
Can do!

Reply:
Shouldn't it just render all taxonomies that have `show_ui` flag to true?
<img width="357" alt="Screenshot 2021-09-17 at 16 27 57" src="https://user-images.githubusercontent.com/237508/133811117-c52096e3-281a-41aa-aef7-dc60fbea1225.png">

I understood that this area would be dynamic.

Reply:
Ah I see - that's a nice catch. We could just use that flag then 👍

Reply:
Going to update it so that we can hide the ui here, but all four taxonomies have `show_ui` as `true`. How do we update that?
<img width="242" alt="Screen Shot 2021-09-17 at 11 59 41 AM" src="https://user-images.githubusercontent.com/22185279/133833370-3eda8821-4ee6-439a-aa57-48dfaa16842d.png">
e